WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginEmailFragment.java

```diff
@@ -130,11 +101,9 @@ public void onClick(View view) {
                 AnalyticsTracker.track(AnalyticsTracker.Stat.LOGIN_GOOGLE_TAPPED);
                 WPActivityUtils.hideKeyboard(getActivity().getCurrentFocus());
 
-                // Start login process.
-                if (NetworkUtils.checkConnection(getActivity()) && !isResolvingError) {
+                if (NetworkUtils.checkConnection(getActivity())) {
                     mOldSitesIDs = SiteUtils.getCurrentSiteIds(mSiteStore, false);
-                    connectGoogleClient();
-                    isSocialLogin = true;
+                    mLoginGoogleFragment = addGoogleFragment();
```

Comment:
If the child fragment is already added, this will silently fail. Example: Tap on the "Log in with Google" button but cancel the process by tapping "BACK". Then tap the "Log in with Google" again, but this time nothing happens.

This probably means we need to handle the child fragment's lifecycle in more detail, perhaps removing the fragment when appropriate.

Reply:
I updated the `addGoogleFragment` method in b5fc2f864e117759519abe2579d2a0cd162507c9.